blocks/footer/__copyright/footer__copyright.css

```diff
@@ -0,0 +1,24 @@
+.footer__copyright {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 153px;
+}
+
+@media screen and (max-width: 1199px) {
+  .footer__copyright {
+    position: relative;
+    width: 130px;
+  }
+}
+
+@media screen and (max-width: 659px){
+  .footer__copyright{
+    display: flex;
```

Comment:
@ElenaPodv It's unnecessary to redefine these Flexbox properties since they are already specified in lines 2 to 4:

`display: flex;`
`align-items: center;`
`justify-content: space-between;`

Reply:
@naumch1k I revisited the code and confirmed that these properties were already specified in lines 2 to 4 so I have removed the redundant Flexbox properties .